content/2_Bronze/Intro_Graphs.mdx

```diff
@@ -74,3 +75,188 @@ Knowing [DFS](/silver/dfs) can be helpful but it should not be required.
 ## Problems
 
 <Problems problems="general" />
+
+## Check Your Understanding
+
+<Quiz>
+  <Quiz.Question>
+    How many connected components are in the following graph?
+		<div height="200">
```

Comment:
by the way this doesn't work. Use [tailwind heights][(https://tailwindcss.com/docs/height) as a class instead (using `className`)

Reply:
whoops yeah I think I forgot to remove that

Reply:
I ended up just manually decreasing the resolution of the images to decrease their size, but let me know if you know of a better way to do so

Reply:
use tailwind heights!

Reply:
it looks like this when i try it
![image](https://user-images.githubusercontent.com/8647920/132960438-f61128be-3333-4ab0-88f3-92b2d32e1185.png)